modules/rostests/apitests/ntdll/precomp.h

```diff
@@ -35,4 +35,9 @@ QuerySetThreadValidator(
     _In_ ULONG InfoLength,
     _In_ NTSTATUS ExpectedStatus);
 
+/* https://www.microsoft.com/resources/msdn/goglobal/default.mspx */
```

Comment:
This URL doesn't work and always say
```
Your request has been blocked. This may be due to several reasons.

1. You are using a proxy that is known to send automated requests to Microsoft. Check with your network administrator if there is any proxy and what User-Agent they are sending in the request header.

2. Your request pattern matches an automated process. To eliminate, reduce the volume of requests over a period of time.

3. Reference ID: 18.2fd47b5c.1621277232.1922a2d0
```

Reply:
Snapshot from 2018: https://web.archive.org/web/20180104073254/https://www.microsoft.com/resources/msdn/goglobal/default.mspx

Reply:
@HBelusca, I get the same text as you when trying to get to the Microsoft website shown.

Reply:
Well, if archive.org has it, let's keep it this way. This URL was already here anyway.